source/src/cip/cipethernetlink.c

```diff
@@ -26,6 +25,9 @@
 #include <stdio.h>
 #include <arpa/inet.h>
 #include <unistd.h>
+
+#define STANDARD_PROTOCOL 0
```

Comment:
There is already a constant for the protocol value 0 defined in ih.h named IPPROTO_IP